docs/topics/secured-mission-sb-interaction.adoc

```diff
@@ -52,6 +68,14 @@ Requesting greeting...
   "content" : "Hello, Scott!"
 }
 ----
++
+Alternatively, call the `/greeting` endpoint directly using the following command:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+curl -kv http://secured-springbot-rest-vx-secured-demo-latest.192.168.42.161.nip.io/greeting?name=Scott -H "Authorization:Bearer $MY_TOKEN"
```

Comment:
Thanks, for catching this. I already updated the examples to use {my-project} instead of the project name. I'll make the IP addresses consistent in the next commit